src/plugins/ini/README.md

```diff
@@ -146,7 +146,7 @@ By default the INI plugin does not support binary data. You can use the [Base64
 
 ```sh
 # Mount INI and recommended plugin Base64
-sudo kdb mount --with-recommends config.ini user/examples/ini ini
+sudo kdb mount config.ini user/examples/ini ini base64
```

Comment:
Doesn't --with-recommends  together with base64 also do the right thing? And we should document that base64 is added to pass ASAN tests.

Reply:
> Doesn't --with-recommends together with base64 also do the right thing?

While this works, using `--with-recommends` is redundant and might also be confusing here.

Reply:
Yes, thus we should explain that "base64" is only there to make ASAN happy.